lib/services/http-service.js

```diff
@@ -220,7 +220,7 @@ function httpServiceFactory(
         );
 
         //re-route common and current
-        var versionPath = configuration.get('versionBase') || '1.1';
+        var versionPath = configuration.get('versionBase') || '2.0';
```

Comment:
I recommend switching to use the standard way we do configurations defaults (which obviously happened after the original code here was written):

`configuration.get('versionBase', '2.0')`